app/subscriber/src/features/todays-front-pages/TodaysFrontPages.tsx

```diff
@@ -50,52 +51,44 @@ export const TodaysFrontPages: React.FC = () => {
   );
 
   React.useEffect(() => {
-    if (!!filter.query.query) {
-      const calendarStartDate = moment(frontPageFilter.startDate).toISOString();
-      const filterStartDate = filter.query.query.bool.must[0].range.publishedOn.gte;
-      if (calendarStartDate !== filterStartDate) {
-        const range = {
```

Comment:
No need to manually do this.  We have shared functions to generate the elastic query.